corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/views.js

```diff
@@ -182,54 +182,12 @@ hqDefine("cloudcare/js/formplayer/apps/views", [
         },
     });
 
-    var LandingPageAppView = Marionette.View.extend({
-        template: _.template($("#landing-page-app-template").html() || ""),
```

Comment:
You can delete https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/cloudcare/templates/cloudcare/partials/grid_view/landing_page_app.html